sources/api/migration/README.md

```diff
@@ -64,15 +60,15 @@ Here's a full example setup for version 1.5:
    -> v1.5.2
    -> v1.5.2_0123456789abcdef
 ```
-The final level has a random identifier appended so we can track migration attempts and help prevent timing issues.
+The migration system appends a random identifier to a the final level so it can track migration attempts and help prevent timing issues.
 
 Old versions can be kept for quick rollbacks, but automated cleanup will be necessary to prevent filling the disk.
 
-Note that the version applies to both the `live` and `pending` trees, which both live inside the directory described above - we have to migrate both live and pending data or we could lose customer information.
+Note that the version applies to both the `live` and `pending` trees, which both live inside the directory described above. Both live and pending data is migrated to prevent user configuration information from being lost.
```

Comment:
Nit: consistency with the new-sentence on a newline pattern.

```suggestion
Note that the version applies to both the `live` and `pending` trees, which both live inside the directory described above.
Both live and pending data is migrated to prevent user configuration information from being lost.
```

Reply:
FWIW, the newline pattern is not universal to all markdown parsers. If we build a website and we want to include files with this pattern they may render is individual paragraphs. 

In this use case, I get it though.